.github/workflows/foreman_plugin.yml

```diff
@@ -85,6 +90,8 @@ jobs:
         include: ${{ fromJson(needs.setup_matrix.outputs.matrix).include }}
         exclude: ${{ fromJson(needs.setup_matrix.outputs.matrix).exclude }}
     steps:
+      - name: Load environment variables from caller
+        run: echo "${{ inputs.environment_variables }}" >> $GITHUB_ENV
```

Comment:
I guess the same should happen in the `existing-database` job? given your example is "handle specific dependencies in your Gemfile" that would also be needed in that branch.